samples/curl/build.gradle

```diff
@@ -12,14 +12,16 @@ repositories {
 def hostPreset = MPPTools.defaultHostPreset(project, [kotlin.presets.macosX64, kotlin.presets.linuxX64, kotlin.presets.mingwX64])
 
 kotlin {
-    targets {
-        fromPreset(hostPreset, 'curl') {
-            compilations.main.outputKinds 'EXECUTABLE'
-            compilations.main.entryPoint 'sample.curl.main'
-
-            if (presets.mingwX64 == hostPreset) {
-                // Add lib path to `libcurl` and its dependencies:
-                compilations.main.linkerOpts "-L${MPPTools.mingwPath()}/lib"
+    targetFromPreset(hostPreset, 'curl') {
+        binaries {
+            executable() {
+                entryPoint = 'sample.curl.main'
+                if (presets.mingwX64 == hostPreset) {
+                    // Add lib path to `libcurl` and its dependencies:
+                    linkerOpts += ["-L${MPPTools.mingwPath()}/lib".toString()]
+                    runTask.environment('PATH': "${MPPTools.mingwPath()}/bin".toString())
```

Comment:
Why toString on String?

Reply:
This is Groovy String, but not Java String, so Kotlin has problems6 because there is no such type.